src/zimscraperlib/inputs.py

```diff
@@ -42,7 +48,8 @@ def handle_user_provided_file(
 
     if str(source).startswith("http"):
         logger.debug(f"download {source} -> {dest}")
-        stream_file(url=str(source), fpath=dest)
+        headers = {"User-Agent": f"{PROJECT_NAME.replace(' ','/')} ({CONTACT})"}
```

Comment:
What's the reasoning for the `/` instead of the space?

Reply:
User-Agent's format is like this "name/version (contact)"
PROJECT_NAME is "name version"

Reply:
This comes from https://meta.wikimedia.org/wiki/User-Agent_policy

It seems to be quite in-line with what MDN considers "common practice": https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
Then we should probably expose a constant with the UA

Reply:
Agreed, in place of the `CONTACT`? I don't think we need both

Reply:
Indeed !

Reply:
I didn't get that. Please elaborate.
`user_agent: Optional[str]=f"{PROJECT_NAME.replace(' ','/')} ({CONTACT})"` as a whole is configurable.

Reply:
You should replace the `CONTACT` constant by a `DEFAULT_USER_AGENT` constant, which computes directly the proper value (not relying on replace operations which are weird / unstable). You should then use this as default user_agent in place of your f-string.

Btw, you should check in the "default" test that the user-agent looks like expected (e.g. starts with "zimscraperlib/" and ends with " (dev@openzim.org)")